src/main/java/org/spongepowered/mod/mixin/entity/player/MixinEntityPlayerMP.java

```diff
@@ -55,7 +55,7 @@
 @NonnullByDefault
 @Mixin(EntityPlayerMP.class)
 @Implements(@Interface(iface = Player.class, prefix = "playermp$"))
-public abstract class MixinEntityPlayerMP extends EntityPlayer implements CommandSource {
+public abstract class MixinEntityPlayerMP extends SpongeUser implements CommandSource {
 
```

Comment:
I was going off a suggestion from [this issue](https://github.com/SpongePowered/Sponge/issues/122). I'm going to close this PR and find a better solution, though.